docs/targets/elasticsearch.md

```diff
@@ -2,47 +2,52 @@
 
 This event Target receives [CloudEvents][ce] over HTTP and writes their payload to an [Elasticsearch][es] index.
 
-## Prerequisites
+## Prerequisite(s)
 
-1. If there is no pre-existing Elasticsearch cluster available, get started quickly by using either [Elastic Cloud on Kubernetes][eck] or [Elastic Cloud][elasticcloud].
+- An Elasticsearch cluster
 
-1. Retrieve credentials for the Elasticsearch cluster in either of the following forms:
+- Credentials for the Elasticsearch cluster in either of the following forms:
     * User and password
     * API Key
 
-Consult the [Elasticsearch documentation][docs] for more information on Elasticsearch.
+Consult the [Secrets](../guides/secrets.md) guide for more information about
+how to add the Elasticsearch password and API key as secrets.
+
+## Creating an Elasticsearch Cluster
+
+Create an Elasticsearch cluster quickly by using either [Elastic Cloud on Kubernetes][eck] or [Elastic Cloud][elasticcloud].
 
 ## Deploying an Instance of the Target
 
 Open the Bridge creation screen and add a Target of type `Elasticsearch`.
 
 ![Adding a Elasticsearch Target](../images/elasticsearch-target/create-bridge-1.png)
 
-In the Target creation form, give a name to the event Target and add the following information:
+In the Target creation form, provide a name for the event Target and add the following information:
 
-* **Index** : Name of the index to write documents to.
-* **Addresses** : List of URLs of Elasticsearch servers.
-* **Skip verify** : Allow skipping the server certificate verification.
-* **CA certificate** : CA certificate to be used by the event target's client, in PEM format.
+* **Index**: Name of the index to write documents to.
+* **Addresses**: List of URLs of Elasticsearch servers.
+* **Skip verify**: Allow skipping of server certificate verification.
+* **CA certificate**: CA certificate to be used by the event Target's client, in PEM format.
 * **Username**: Elasticsearch username.
-* **Password**:  Reference a [TriggerMesh secret](../guides/secrets.md) containing a password to communicate with the Elasticsearch API, as described in the previous section.
-* **API key**: Reference a [TriggerMesh secret](../guides/secrets.md) containing an API token to communicate with the Elasticsearch API, as described in the previous section.
+* **Password**:  Reference a [TriggerMesh secret](../guides/secrets.md) containing a password to communicate with the Elasticsearch API, as discussed in the prerequisites.
+* **API key**: Reference a [TriggerMesh secret](../guides/secrets.md) containing an API token to communicate with the Elasticsearch API, as discussed in the prerequisites.
```

Comment:
Same as above - should we link to the section? I am wondering about this. Since we are changing the location on how to create Keys should we link to the credential section? What do you think?